api/client/samples/crop_w_series/top_5_crop_w_series.ipynb

```diff
@@ -0,0 +1,1415 @@
+{
```

Comment:
<p>I think it will be helpful to print the country name when you print out There are xxx regions for the selected country/level</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>